pkg/app/cmd.go

```diff
@@ -199,6 +200,7 @@ func (o *options) bindFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.up, "up", false, "provision the test cluster")
 	flags.BoolVar(&o.down, "down", false, "tear down the test cluster")
 	flags.StringVar(&o.test, "test", "", "test type to run, if unset no tests will run")
+	flags.BoolVar(&o.skipTestJUnitReport, "skip-test-junit-report", false, "skip reporting the test result as a JUnit test case")
```

Comment:
Is "test" a necessary distinction here? 

Vs "skip-junit-report", "skip reporting kubetest2 steps to junit.xml", etc

Reply:
IIUC the intention is to only skip adding the junit entry for the `kubetest2.Test()` step. build, up, down will still show.
It's not too ideal but since we have explicit tester binaries I can see use cases where it should be treated specially.

Reply:
Thanks @amwat . Yes, that is the intention for this change.